Improvement/zenko 4616 mongodb 5.0

williamlardier opened the pull request:
To be merged when we have dedicated branch / pensieve API mongodb driver migration merged and released.

The PR is about upgrading the mongodb charts to support v5.0. There is also a need to update the init scripts for the shard server https://github.com/scality/Zenko/pull/1913/commits/3703a948ba7658950544ac2bf542f1791da44b4d

bert-e commented:
# Hello williamlardier,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://bert-e.scality.net/scality/zenko/doc/user).





*Status report is not available.*

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue ZENKO-4616 contains:


* *None*



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `2.8.1`


Please check the `Fix Version/s` of ZENKO-4616, or the target
branch of this pull request.

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue ZENKO-4616 contains:


* *None*



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `2.8.3`


Please check the `Fix Version/s` of ZENKO-4616, or the target
branch of this pull request.

bert-e commented:
# Branches have diverged



This pull request's source branch `improvement/ZENKO-4616-mongodb-5.0` has diverged from
`development/2.8` by more than 50 commits.

To avoid any integration risks, please re-synchronize them using one of the
following solutions:

* Merge `origin/development/2.8` into `improvement/ZENKO-4616-mongodb-5.0`
* Rebase `improvement/ZENKO-4616-mongodb-5.0` onto `origin/development/2.8`

Note: If you choose to rebase, you may have to ask me to rebuild
integration branches using the `reset` command.

Seragonia committed:
ZENKO-4616: ease debugging in CI with more logs

Seragonia committed:
ZENKO-4616: bump mongo to v5

Seragonia committed:
ZENKO-4616: support latest mongodb init scripts for shardsvr

Seragonia committed:
ZENKO-4616: upgrade charts to v5.0

Seragonia committed:
ZENKO-4616: bump pensieve API

The new pensieve API now implements the official
mongodb driver. This ensure compatibility with
mongodb v5.

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue ZENKO-4616 contains:


* *None*



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `2.8.5`


Please check the `Fix Version/s` of ZENKO-4616, or the target
branch of this pull request.

francoisferrand commented on VERSION:
```diff
@@ -1,4 +1,4 @@
-VERSION="2.8.4"
+VERSION="2.9.0"
```
should not target 2.9 branch, do not forget to create it and retarget PR before merging.

(though we should not release zenko 2.9.0 right after : if you need to create a release afterwards for bumping in Artesca, should be a pre-release like `2.9.0-preview.1` or `2.9.0-alpha.1`)

williamlardier replied:
Yes, this is only for testing.

williamlardier replied:
PR is now rebased on the latest dev/2.9 branch.

nicolas2bert commented on solution/deps.yaml:
```diff
@@ -47,9 +47,9 @@ kafka-lag-exporter:
   envsubst: KAFKA_LAGEXPORTER_TAG
 pensieve-api:
-  sourceRegistry: registry.scality.com/pensieve-api
+  sourceRegistry: registry.scality.com/playground/wlardier
```
Reminder: don't forget to update it :)

williamlardier replied:
Updated and branch rebased to dev/2.9

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue ZENKO-4616 contains:


* *None*



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `2.8.6`


Please check the `Fix Version/s` of ZENKO-4616, or the target
branch of this pull request.

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue ZENKO-4616 contains:


* *None*



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `2.8.10`


Please check the `Fix Version/s` of ZENKO-4616, or the target
branch of this pull request.

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue ZENKO-4616 contains:


* *None*



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `2.8.18`

* `2.9.0`


Please check the `Fix Version/s` of ZENKO-4616, or the target
branch of this pull request.

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue ZENKO-4616 contains:


* *None*



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `2.9.0`


Please check the `Fix Version/s` of ZENKO-4616, or the target
branch of this pull request.

Seragonia committed:
ZENKO-4616: bump zenko version

ghost commented:
/approve

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue ZENKO-4616 contains:


* *None*



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `2.9.0`


Please check the `Fix Version/s` of ZENKO-4616, or the target
branch of this pull request.




*The following options are set:* **approve**

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/2.9`



The following branches will **NOT** be impacted:

* `development/2.5`
* `development/2.6`
* `development/2.7`
* `development/2.8`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/2.9`



The following branches have **NOT** changed:

* `development/2.5`
* `development/2.6`
* `development/2.7`
* `development/2.8`



Please check the status of the associated issue ZENKO-4616.

Goodbye williamlardier.
